sphinx/source/docs/user_guide/graphs.rst

```diff
@@ -0,0 +1,202 @@
+.. _userguide_graphs:
+
+Visualizing Network Graphs
+==========================
+
+Bokeh has added native support for creating network graph visualizations with
```

Comment:
"has added" appropriate for announcement, docs should just say "Bokeh supports.."